central/convert/storagetov2/vuln_exception_service.go

```diff
@@ -31,8 +31,8 @@ func VulnerabilityException(vulnRequest *storage.VulnerabilityRequest) *v2.Vulne
 		TargetState: convertVulnerabilityState(vulnRequest.GetTargetState()),
 		Status:      exceptionStatus(vulnRequest.GetStatus()),
 		Expired:     vulnRequest.GetExpired(),
-		Requester:   convertUser(vulnRequest.GetRequestor()),
-		Approvers:   convertUsers(vulnRequest.GetApprovers()),
+		Requester:   requester(vulnRequest.GetRequesterV2()),
```

Comment:
This caused me to go down a rabbit hole of "requester" vs "requestor".  I learned 2 things:
1. I concur with the change
2. English is painful